tests/configs/fetch.config.ts

```diff
@@ -178,4 +178,15 @@ export default defineConfig({
       target: '../specifications/parameters.yaml',
     },
   },
+  noIndexFiles: {
```

Comment:
Since `indexFiles` is a global property, not limited to `fetch`, I think it is appropriate to write the test case in `default`.

Reply:
Oh right, I have moved it to `default`, please see new commit!